Lab1.ipynb

```diff
@@ -241,15 +249,256 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# 1A-B:\n",
+    "n = 10\n",
+    "p = 0.5\n",
+    "# defining list of r values \n",
+    "r_values = list(range(n + 1)) \n",
+    "# list of pmf values \n",
+    "dist = [binom.pmf(r, n, p) for r in r_values ] \n",
+    "# plotting the graph\n",
+    "plt.bar(r_values, dist) \n",
+    "plt.xlabel('X', fontsize = 27) \n",
+    "plt.ylabel('P(x)', fontsize = 27)\n",
+    "plt.title('Bionomial Distribution w/ n = 10, p = .5', fontsize = 30)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Above shows with n trials and the probablity of success being 50% what is the spread across those trials. You can see since it's a 50/50 shot, the spread is mirrored across the medium which is 5. "
```

Comment:
Part 2 Problem 1B:
- What do you mean "probability of success being 50% what is spread across those trials"? Are we interpreting the most probable number of events to be 5 from the p value of p=0.5?
- I think the wording is more along the lines of "the probability of x events for n=10 trials when p=0.5" (from the Lecture 3 notes)